2024-08-31 22:39:14,745 - __main__ - INFO - Starting the blockchain system...
2024-08-31 22:39:14,746 - __main__ - INFO - Configuration loaded. Local Host: 127.0.0.1, Local Host Port: 8888, Web Port: 5900
2024-08-31 22:39:15,088 - __main__ - INFO - Web app process started.
2024-08-31 22:39:15,089 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 8888, isNewBlockAvailable: False
2024-08-31 22:39:15,091 - __main__ - INFO - Sync server process started.
2024-08-31 22:39:15,094 - __main__ - INFO - Starting sync process...
2024-08-31 22:39:15,094 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:39:15,094 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:39:15,094 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:39:15,094 - Blockchain.Backend.core.network.syncManager - INFO - Starting download from node at port 5555...
2024-08-31 22:39:15,095 - Blockchain.Backend.core.network.syncManager - INFO - Last block found with header: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,095 - Blockchain.Backend.core.network.syncManager - DEBUG - Requesting headers starting from block: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,096 - Blockchain.Backend.core.network.syncManager - ERROR - Exception occurred in startDownload: [Errno 111] Connection refused
Traceback (most recent call last):
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 178, in startDownload
    self.connectToHost(localport, port, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 151, in connectToHost
    self.socket = self.connect.connect(localport, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/connection.py", line 27, in connect
    self.socket.connect((self.host, self.port))
ConnectionRefusedError: [Errno 111] Connection refused
2024-08-31 22:39:15,097 - __main__ - INFO - Sync with node at port 5555 completed successfully
2024-08-31 22:39:15,097 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:39:15,097 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:39:15,097 - Blockchain.Backend.core.network.syncManager - INFO - Starting download from node at port 3333...
2024-08-31 22:39:15,098 - Blockchain.Backend.core.network.syncManager - INFO - Last block found with header: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,099 - Blockchain.Backend.core.network.syncManager - DEBUG - Requesting headers starting from block: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,099 - Blockchain.Backend.core.network.syncManager - ERROR - Exception occurred in startDownload: [Errno 111] Connection refused
Traceback (most recent call last):
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 178, in startDownload
    self.connectToHost(localport, port, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 151, in connectToHost
    self.socket = self.connect.connect(localport, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/connection.py", line 27, in connect
    self.socket.connect((self.host, self.port))
ConnectionRefusedError: [Errno 111] Connection refused
2024-08-31 22:39:15,099 - __main__ - INFO - Sync with node at port 3333 completed successfully
2024-08-31 22:39:15,099 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:39:15,099 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:39:15,099 - Blockchain.Backend.core.network.syncManager - INFO - Starting download from node at port 1111...
2024-08-31 22:39:15,100 - Blockchain.Backend.core.network.syncManager - INFO - Last block found with header: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,101 - Blockchain.Backend.core.network.syncManager - DEBUG - Requesting headers starting from block: 000054ec2ee91e00b00c016cdd508085aa684d02f062b9ab9458d4eda8f4a065
2024-08-31 22:39:15,101 - Blockchain.Backend.core.network.syncManager - ERROR - Exception occurred in startDownload: [Errno 111] Connection refused
Traceback (most recent call last):
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 178, in startDownload
    self.connectToHost(localport, port, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/syncManager.py", line 151, in connectToHost
    self.socket = self.connect.connect(localport, bindPort)
  File "/home/oxana/Dropbox/study/MedSchool/AU/COMP498/blockchain-udemy-adv/Blockchain/Backend/core/network/connection.py", line 27, in connect
    self.socket.connect((self.host, self.port))
ConnectionRefusedError: [Errno 111] Connection refused
2024-08-31 22:39:15,101 - __main__ - INFO - Sync with node at port 1111 completed successfully
2024-08-31 22:39:15,499 - werkzeug - INFO -  * Running on http://127.0.0.1:5900/ (Press CTRL+C to quit)
2024-08-31 22:39:15,861 - Blockchain.Backend.core.network.syncManager - INFO - SERVER STARTED
2024-08-31 22:39:15,862 - Blockchain.Backend.core.network.syncManager - INFO - [LISTENING] at 127.0.0.1:8888
2024-08-31 22:39:15,862 - Blockchain.Backend.core.network.syncManager - DEBUG - Waiting to accept a connection...
2024-08-31 22:50:55,984 - __main__ - INFO - Starting sync process...
2024-08-31 22:50:55,984 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:50:55,984 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:50:55,984 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:50:55,985 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:50:55,985 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:50:55,985 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:50:55,985 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:50:55,985 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:50:55,986 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:50:55,986 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:04,104 - __main__ - INFO - Starting sync process...
2024-08-31 22:51:04,104 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:51:04,104 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:51:04,104 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:51:04,105 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:04,105 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:51:04,105 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:51:04,105 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:04,105 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:51:04,105 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:51:04,105 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:09,143 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:09] "GET /wallet HTTP/1.1" 200 -
2024-08-31 22:51:09,173 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:09] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:09,174 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:09] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:09,175 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:09] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:24,138 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:24] "POST /wallet HTTP/1.1" 200 -
2024-08-31 22:51:24,154 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:24] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:24,157 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:24] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:24,158 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:24] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:26,097 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:26] "GET /mempool HTTP/1.1" 200 -
2024-08-31 22:51:26,111 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:26] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:26,118 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:26] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:26,119 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:26] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:27,085 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:27] "GET /memTx/d8eec970cbf984e827653fbacc6913ab4bdff84e31d89633f0c16acc1a1c58e3 HTTP/1.1" 200 -
2024-08-31 22:51:27,097 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:27] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:27,100 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:27] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:27,101 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:27] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:31,148 - __main__ - INFO - Starting sync process...
2024-08-31 22:51:31,148 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:51:31,148 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:51:31,148 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:51:31,149 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:31,149 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:51:31,149 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:51:31,149 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:31,149 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:51:31,149 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:51:31,149 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:44,148 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:44] "GET /transactions HTTP/1.1" 200 -
2024-08-31 22:51:44,159 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:44] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:44,163 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:44] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:44,163 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:44] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:45,296 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:45] "GET /mempool HTTP/1.1" 200 -
2024-08-31 22:51:45,308 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:45] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:45,312 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:45] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:45,314 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:45] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:46,226 - __main__ - INFO - Starting sync process...
2024-08-31 22:51:46,226 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:51:46,226 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:51:46,226 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:51:46,227 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:46,227 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:51:46,227 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:51:46,227 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:46,227 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:51:46,227 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:51:46,227 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:46,794 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:46] "GET /memTx/d8eec970cbf984e827653fbacc6913ab4bdff84e31d89633f0c16acc1a1c58e3 HTTP/1.1" 200 -
2024-08-31 22:51:46,806 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:46] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:46,809 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:46] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:46,809 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:46] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:52,625 - __main__ - INFO - Starting sync process...
2024-08-31 22:51:52,625 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:51:52,625 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:51:52,626 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:51:52,626 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:52,626 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:51:52,626 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:51:52,626 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:52,626 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:51:52,626 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:51:52,627 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:57,616 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:57] "GET /mempool HTTP/1.1" 200 -
2024-08-31 22:51:57,626 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:57] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:51:57,630 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:57] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:57,631 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:51:57] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:51:59,906 - __main__ - INFO - Starting sync process...
2024-08-31 22:51:59,906 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:51:59,906 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:51:59,906 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:51:59,907 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:59,907 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:51:59,907 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:51:59,907 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:51:59,907 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:51:59,907 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:51:59,908 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:01,802 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:01] "GET /transactions HTTP/1.1" 200 -
2024-08-31 22:52:01,816 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:01] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:01,818 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:01] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:01,821 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:01] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:03,680 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:03] "GET /block HTTP/1.1" 200 -
2024-08-31 22:52:03,695 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:03] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:03,697 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:03] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:03,699 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:03] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:05,545 - __main__ - INFO - Starting sync process...
2024-08-31 22:52:05,545 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:52:05,545 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:52:05,545 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:52:05,545 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:05,545 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:52:05,545 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:52:05,546 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:05,546 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:52:05,546 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:52:05,546 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:11,517 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:11] "[32mGET /block?blockHeader=000000844aa8aaa1bb04fa5848a29f199621184ab27d678bfb45612863ab0721 HTTP/1.1[0m" 302 -
2024-08-31 22:52:11,578 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:11] "GET /block/000000844aa8aaa1bb04fa5848a29f199621184ab27d678bfb45612863ab0721 HTTP/1.1" 200 -
2024-08-31 22:52:11,592 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:11] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:11,594 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:11] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:11,596 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:11] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:20,965 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:20] "[32mGET /transactions/94368ad5052fa9fa65f21a83acf0e16001968084514947ef1b2189d06fd47ea0 HTTP/1.1[0m" 302 -
2024-08-31 22:52:20,982 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:20] "GET /tx/94368ad5052fa9fa65f21a83acf0e16001968084514947ef1b2189d06fd47ea0 HTTP/1.1" 200 -
2024-08-31 22:52:21,026 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:21] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:21,030 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:21] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:21,032 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:21] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:28,119 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:28] "GET /mempool HTTP/1.1" 200 -
2024-08-31 22:52:28,136 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:28] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:28,141 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:28] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:28,142 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:28] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:29,394 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:29] "GET /transactions HTTP/1.1" 200 -
2024-08-31 22:52:29,406 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:29] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:29,410 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:29] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:29,410 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:29] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:30,742 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:30] "GET /block HTTP/1.1" 200 -
2024-08-31 22:52:30,755 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:30] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:30,758 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:30] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:30,760 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:30] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:31,400 - __main__ - INFO - Starting sync process...
2024-08-31 22:52:31,400 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:52:31,400 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:52:31,401 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:52:31,401 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:31,401 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:52:31,401 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:52:31,401 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:31,401 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:52:31,401 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:52:31,401 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:32,834 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:32] "[32mGET /block?blockHeader=00007ccc830819406d9102ba53ba9cb7f75b3121a33fe694fa386b2a1d858f95 HTTP/1.1[0m" 302 -
2024-08-31 22:52:32,854 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:32] "GET /block/00007ccc830819406d9102ba53ba9cb7f75b3121a33fe694fa386b2a1d858f95 HTTP/1.1" 200 -
2024-08-31 22:52:32,870 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:32] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:32,873 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:32] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:32,874 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:32] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:36,196 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:36] "[32mGET /transactions/9de324f1841d76bfb5ea5da597a515a01b98db5cf634d110c909a3ae036448dc HTTP/1.1[0m" 302 -
2024-08-31 22:52:36,213 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:36] "GET /tx/9de324f1841d76bfb5ea5da597a515a01b98db5cf634d110c909a3ae036448dc HTTP/1.1" 200 -
2024-08-31 22:52:36,243 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:36] "[36mGET /static/css/style.css HTTP/1.1[0m" 304 -
2024-08-31 22:52:36,246 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:36] "[36mGET /static/images/CODIES_ALERT.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:36,250 - werkzeug - INFO - 127.0.0.1 - - [31/Aug/2024 22:52:36] "[36mGET /static/images/search.png HTTP/1.1[0m" 304 -
2024-08-31 22:52:40,278 - __main__ - INFO - Starting sync process...
2024-08-31 22:52:40,278 - __main__ - DEBUG - Retrieved port list from NodeDB: [8888, 5555, 3333, 1111]
2024-08-31 22:52:40,279 - __main__ - INFO - Syncing with node at port 5555
2024-08-31 22:52:40,279 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 5555, isNewBlockAvailable: True
2024-08-31 22:52:40,279 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:40,279 - __main__ - INFO - Syncing with node at port 3333
2024-08-31 22:52:40,279 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 3333, isNewBlockAvailable: True
2024-08-31 22:52:40,279 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
2024-08-31 22:52:40,279 - __main__ - INFO - Syncing with node at port 1111
2024-08-31 22:52:40,279 - Blockchain.Backend.core.network.syncManager - DEBUG - Constructor: syncManager initialized with host: 127.0.0.1, port: 1111, isNewBlockAvailable: True
2024-08-31 22:52:40,279 - __main__ - ERROR - Error while publishing or Downloading a Blockchain
[Errno 111] Connection refused
